### DISCUSSION TITLE
Use PuppyGit sync your markor notes on Android

### DISCUSSION THREAD

**Discussion by Bandeapart1964** [gsantner/markor#2457] [Show and tell]
Hi, I wrote a git client for android, it can be used to sync your markor notes on android devices, is free and no ads, welcome try:
https://github.com/Bandeapart1964/PuppyGit 

 a simple demo video: https://www.patreon.com/posts/markor-puppygit-114681068

**Comment by clach04** [comment]
Cool, I'll be interested in checking it out once it has ssh support.

**Reply by Bandeapart1964** [reply]
> Cool, I'll be interested in checking it out once it has ssh support.

it will support ssh in next version, and support view file history versioned by git, I think git client + notes app is great, only one problem is most users don't know how to use git 😂
